services/dbService.tsx

```diff
@@ -1,53 +1,64 @@
-import { db } from '../firebase/firebase.config';
-import { collection, addDoc, getDocs, QuerySnapshot, DocumentData } from "firebase/firestore"; 
-import { GeoPoint } from 'firebase/firestore';
+import {db} from '../firebase/firebase.config';
+import {collection, addDoc, getDocs, getDoc, doc} from "firebase/firestore";
 import Event from '../models/event';
 
 
 export class EventDoc {
```

Comment:
interface zamiast class. Also może przerzucić to do innego pliku (?)

Reply:
W sumie jest event i jego można używać